juju-charms/charm-trilio-data-mover-api/src/reactive/dmapi_handlers.py

```diff
@@ -115,13 +114,14 @@ def setup_database(database):
     """On receiving database credentials, configure the database on the
     interface.
     """
-    database.configure('nova', 'dmapi', prefix='nova')
-    database.configure('nova_api', 'dmapi', prefix='novaapi')
+    database.configure('nova', 'nova')
+    database.configure('nova_api', 'nova')
```

Comment:
@javacruft , I had to remove the prefix attribute as I was not able to populate the connection url. I followed the model that 'designate' charm is using which was suggested by @ryan-beisner but could not find the solution. As of now, this implementation works just fine for us.